styles/style.css

```diff
@@ -1,31 +1,89 @@
+.container {
+  width: 100%;
+  margin: 0 auto;
+}
 h1 {
   text-align: center;
+  margin-top: 120px;
+  font-family: roboto;
+  color: #383838;
 }
 hr {
   border: 0;
   background-color: #f5bd41;
   width: 100px;
   height: 4px;
 }
-/* TEXT */
-.text {
+.features {
   display: flex;
-  justify-content: center;
-  margin-top: 20px;
-  margin-right: 40px;
+  margin: 50px 0;
 }
-.text_item {
+.features_item {
   width: 20%;
+  text-align: center;
+  padding: 0 65px;
+  margin: 25px 0;
+  font-family: roboto;
+  line-height: 28px;
+}
+.features_icon {
+  margin-bottom: 50px;
+}
+.features_title {
   font-size: 24px;
   color: #383838;
 }
-/* ABOUT */
-.about {
+#lol {
+  margin-bottom: 67px;
+}
+
+@font-face {
+  font-family: roboto;
+  src: url("../fonts/roboto/Roboto-Light.ttf") format("truetype");
+  font-style: normal;
+  font-weight: normal;
+}
+.footer {
+  background-color: #10367c;
+}
+.footer_inner {
   display: flex;
-  justify-content: center;
-  margin-top: 80px;
+  flex-wrap: wrap;
+    padding: 75px;
 }
-.about_img {
-  width: 20%;
-  padding: 20px 0;
+.footer_block {
+  width: 33.33333%;
+  text-align: left;
+}
+.footer_title {
```

Comment:
It's too different, please fix
<img width="506" alt="Screenshot 2022-03-11 at 22 11 19" src="https://user-images.githubusercontent.com/25488411/157935964-acab25b5-d9c3-4834-b685-3181d31f6c10.png">